src/web/components/BusinessContacts/BusinessContactDialog.tsx

```diff
@@ -21,7 +20,12 @@ function BusinessContactDialog({
 }: BusinessContactDialogProps) {
   const [open, setOpen] = useState(false);
 
-  const onSubmit: SubmitHandler<BusinessContactForm> = async (formData) => {
+  const formMethods = useForm<BusinessContactForm>({
+    defaultValues: contact as BusinessContactForm,
```

Comment:
I don't think the `as BusinessContactForm` is needed - and I'm always wary of `as` because you lose type safety.